cli/tests/unit/os_test.ts

```diff
@@ -177,3 +177,9 @@ unitTest({ perms: { env: false } }, function releasePerm(): void {
     Deno.osRelease();
   }, Deno.errors.PermissionDenied);
 });
+
+unitTest({ perms: { env: true } }, function memoryInfo(): void {
+  /* const info = Deno.memoryInfo(); */
+  /* assert(info.total >= 0); */
```

Comment:
@humancalico please uncomment this assertion

Reply:
Done, @bartlomieju